compiler/rustc_error_codes/src/error_codes/E0462.md

```diff
@@ -0,0 +1,31 @@
+Found staticlib `..` instead of rlib or dylib.
+
+Consider the following two files:
+
+`a.rs`
+```ignore (cannot-link-with-other-tests)
+#![crate_type = "staticlib"]
+
+fn foo() {}
+```
+
+`main.rs`
+```ignore (cannot-link-with-other-tests)
+extern crate a;
+
+fn main() {
+    a::foo();
+}
+```
+
+Crate `a` is compiled as a staticlib. A staticlib is a system-dependant library
```

Comment:
Is "staticlib" a real world? First time I see it.

Reply:
Umm, its the one used in the Rustc book (https://doc.rust-lang.org/reference/linkage.html). We could put backticks around it, but that wouldn't be consistent with other Rust docs.

Reply:
Let's do it then. :)